examples/auto-ads.amp.html

```diff
@@ -3,9 +3,10 @@
   <head>
     <meta charset="utf-8">
     <title>AMP #0</title>
-    <link rel="canonical" href="amps.html" >
+    <link rel="canonical" href="http://amp.autoplaced.com" >
     <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
     <script async custom-element="amp-auto-ads" src="https://cdn.ampproject.org/v0/amp-auto-ads-0.1.js"></script>
+    <script async custom-element="amp-sticky-ad" src="https://cdn.ampproject.org/v0/amp-sticky-ad-1.0.js"></script>
```

Comment:
amp-ad.js will be auto loaded if not included in <head> yet. That is why this test page work. But apparently it is not allowed to auto load <amp-sticky-ad> in the same way.